thoth/storages/graph/postgres.py

```diff
@@ -1767,6 +1767,43 @@ def get_si_unanalyzed_python_package_versions_count_all(
 
             return query.count()
 
+    def _construct_get_solver_query(
+        self,
+        session: Session
+    ) -> Query:
+        """Construct query to retrieve solvers."""
+        query = (
+            session.query(EcosystemSolver)
+            .filter(EcosystemSolver.os_name != 'rhel')   # Thoth considers rhel == ubi
```

Comment:
Why is this needed? We should prevent syncing data if they introduce RHEL vs UBI incosystency. Data in the database should be normalized and consistent.

Reply:
> Why is this needed? We should prevent syncing data if they introduce RHEL vs UBI incosystency. Data in the database should be normalized and consistent.

we didn't do it immediately, so we have both rhel and ubi, and we have methods to map_os_name when you use query actually ubi8-py38 is missing, hence issue opened

Reply:
> > Why is this needed? We should prevent syncing data if they introduce RHEL vs UBI incosystency. Data in the database should be normalized and consistent.
> 
> we didn't do it immediately, so we have both rhel and ubi, and we have methods to map_os_name when you use query actually ubi8-py38 is missing, hence issue opened.

Reply:
I think keeping this inconsistency might be not a good idea long-term. Can we create a database migration to fix data consistency?

Reply:
> I think keeping this inconsistency might be not a good idea long-term. Can we create a database migration to fix data consistency?

agree it is not good. But we need to remove data and resync them, so that there is no in ecosystem solver, as we do not have a rhel solver, but only a ubi solver and the bug we are solving now on the sync side need to be used for re syncing

Reply:
I think a simple UPDATE SQL query that would work on os_name = rhel/ubi should work fine. Not to remove data which might cause issues with already linked entries in the database.

Reply:
We do this directly on the database then, thanks @fridex, something more on this PR? :)

Reply:
I think the rhel string should not be placed here. If we decide solvers `rhel==ubi` are equivalent, we should update the data in the database to keep them consistent following https://github.com/thoth-station/storages/issues/2199

Reply:
Adjusted in dc0267a8d1c9e33e5b3d6909ccda917bd6fb8487, Thanks!